traitsui/tests/editors/test_range_editor_text.py

```diff
@@ -66,6 +78,31 @@ def test_wx_spin_control_editing():
     assert num.number >= 3 and num.number <= 8
 
 
+@skip_if_not_qt4
+def test_avoid_slider_feedback():
```

Comment:
Can we please include these two functions in a unittest.TestCase?

Reply:
Not when adding to an existing nose-based suite.  At some point it would be good to switch to unittest-based only, but it should be a stand-alone PR.